Makefile

```diff
@@ -91,6 +91,7 @@ fmt-md:
 .PHONY: generate
 generate:
 	go generate ./...
+	mockgen -source=pkg/ingestor/parser/common/types.go -destination=pkg/ingestor/parser/internal/mock/types.go -package=mocks -copyright_file=pkg/ingestor/parser/internal/mock/license.txt
```

Comment:
This can be moved to a `mockgen.go` similar to what we did here for the [scorecard mock gen](https://github.com/guacsec/guac/blob/main/internal/testing/mocks/mockgen.go). So that when `go generate ./...` is called, it will run the proper mockgen.

Reply:
👍